README.md

```diff
@@ -17,3 +17,14 @@ Here is how you make a test deployment:
     $ vim instance/settings.py # Customize this file as needed
     $ pip install -r requirements.txt
     $ python runserver.py
+
+
+## Asset builds
+
+To build for production:
+
+```
+    cd hgtv/assets
+    npm install
+    npm run build
```

Comment:
Can this be moved to a `make` command?

Reply:
If there isn't make involved in the project already, why involve now.